indigo/distribution.yaml

```diff
@@ -1964,6 +1964,23 @@ repositories:
       url: https://github.com/ipa320/cob_supported_robots.git
       version: indigo_dev
     status: developed
+  cogrob_catkin_grpc:
+    doc:
+      type: git
+      url: https://github.com/CogRobRelease/catkin_grpc-release.git
+      version: master
```

Comment:
The master branch of the release repo doesn't contain any ROS packages which could be indexed. You probably want to use the source repo here.

Reply:
Thanks for the comments. PTAL.